lib/redux/slices/drawer.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { DrawerComponentProps } from '@/utils/types/redux'
+
+const initialState: DrawerComponentProps = {
+    drawerDisplay: '',
+    showDrawer: false
+}
+
+export const drawerComponentSlice = createSlice({
+    name: 'drawerComponent',
+    initialState,
+    reducers: {
+        showDrawer: (state, { payload: { drawerDisplay } }) => {
+            state.drawerDisplay = drawerDisplay
```

Comment:
drawerDisplay -> payload. No longer sending an object bc there is only one item in the payload so the item is the entire payload